ironfish/src/memPool/memPool.ts

```diff
@@ -46,22 +62,25 @@ export class MemPool {
 
   constructor(options: {
     chain: Blockchain
+    consensus: Consensus
     feeEstimator: FeeEstimator
     metrics: MetricsMonitor
+    maxSizeBytes: number
+    recentlyEvictedCacheSize: number
     logger?: Logger
   }) {
     const logger = options.logger || createRootLogger()
 
+    this.maxSizeBytes = options.maxSizeBytes
+    this.consensus = options.consensus
     this.head = null
 
-    this.queue = new PriorityQueue<MempoolEntry>(
-      (firstTransaction, secondTransaction) => {
-        if (firstTransaction.feeRate !== secondTransaction.feeRate) {
-          return firstTransaction.feeRate > secondTransaction.feeRate
-        }
+    this.queue = new PriorityQueue<MempoolEntry>(compareMempoolEntries, (t) =>
```

Comment:
as there are multiple queues now, can `this.queue` be named into something more indicative of what it does?i.e. `this.toBlockQueue` or something

Reply:
Yea I like that. Changing to `feeRateQueue`